src/components/AnonymousChat/PostNotificationChannelItem.tsx

```diff
@@ -101,6 +103,12 @@ const PostNotificationChannelItem: (props: MessageChannelItemProps) => React.Rea
       unreadCount={postNotifItem?.unreadCount}
       upvote={item?.rawJson?.upvote}
       channelType={item?.channelType}
+      dbAnonUserInfo={{
+        anon_user_info_color_code: item?.anon_user_info_color_code,
+        anon_user_info_color_name: item?.anon_user_info_color_name,
+        anon_user_info_emoji_code: item?.anon_user_info_emoji_code,
+        anon_user_info_emoji_name: item?.anon_user_info_emoji_name
+      }}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code adds a `dbAnonUserInfo` object to the props passed to `BaseChannelItem`. This could potentially change the behavior of the `BaseChannelItem` component if it uses this prop. Please ensure that the `BaseChannelItem` component correctly handles this new prop.
```diff
+  dbAnonUserInfo={{
+    anon_user_info_color_code: item?.anon_user_info_color_code,
+    anon_user_info_color_name: item?.anon_user_info_color_name,
+    anon_user_info_emoji_code: item?.anon_user_info_emoji_code,
+    anon_user_info_emoji_name: item?.anon_user_info_emoji_name
+  }}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->